packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -36,9 +37,10 @@ export interface Props {
   noHeader?: boolean;
   resizable?: boolean;
   initialSortBy?: TableSortByFieldState[];
-  onCellClick?: TableFilterActionCallback;
+  onCellClick?: TableCellClickActionCallback;
   onColumnResize?: TableColumnResizeActionCallback;
   onSortByChange?: TableSortByActionCallback;
+  onFilterAdded?: TableFilterActionCallback;
```

Comment:
This feels a bit too domain-specific as for a generic table component

Reply:
Not sure what the solution would be otherwise? Maybe we could sync on this or maybe you could elaborate further?